src/Microsoft.DotNet.XHarness.Tests.Runners/ApplicationEntryPoint.cs

```diff
@@ -163,6 +165,30 @@ private async Task<List<string>> GetIgnoredCategories()
             return categories;
         }
 
+        internal static void ConfigureRunner(TestRunner runner, ApplicationOptions options)
+        {
+            runner.RunAllTestsByDefault = options.RunAllTestsByDefault;
+        }
+
+        internal static string WriteResults(TestRunner runner, ApplicationOptions options, LogWriter logger, TextWriter writer = null)
+        {
+            TestRunner.Jargon jargon = options.XmlVersion.ToTestRunnerJargon();
+            string resultsFilePath = null;
+
+            if (options.EnableXml)
+            {
+                runner.WriteResultsToFile(writer ?? Console.Out, jargon);
+                logger.Info("Xml file was written to the tcp listener.");
```

Comment:
Should it say "to console"? I think this might help is with investigations

Reply:
Will do